index.html

```diff
@@ -49,27 +62,42 @@ <h1>I’m Alexandra. <br />Glad to see you!</h1>
                 </div>
                 <div class="card-description">
                     <div>
-                        <h3 class="card-title">Tonic</h3>
+                        <h3 class="card-title mobile">Tonic</h3>
+                        <h3 class="card-title desktop">Facebook 360</h3>
```

Comment:
- [ ] **[OPTIONAL]** Please consider to use the correct background-color for the project section so it match the figma design. 
**Your project design**
![image](https://user-images.githubusercontent.com/74411135/229304742-138d9442-1171-4f7c-86ab-276d2a9448c8.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229304791-5339f200-d474-49e4-ad37-036030209326.png)

- [ ] **[OPTIONAL]** Please consider adding also the border-raduis on relevant section (headline section and about section 

**Figma design: headline section**
![image](https://user-images.githubusercontent.com/74411135/229304913-80a34de2-20b9-435d-a9f8-6ad41cf9d02b.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229304880-a3d3f65d-a18f-4220-94d6-495abb70b650.png)


**Note:** This project requirement was missed by the previous reviewer so please consider implementing it.

Reply:
Sure, dear @Trast00 ... I'll do it during the weekend. And yes, nobody told me so far that issue, brother. I really appreciate your time and where is exactly my mistake. I'll fix that detail to begin the next week with my project ready to updates on Week 3 🙂🙂👏👏🙏🙏